rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -737,7 +737,7 @@ namespace rsx
 	*/
 	void thread::fill_vertex_program_constants_data(void *buffer)
 	{
-		memcpy(buffer, rsx::method_registers.transform_constants.data(), 468 * 4 * sizeof(float));
```

Comment:
I'd like to avoid uploading inaccessible registers if possible. We need proof that they actually get loaded on realhw and not just return 0 when reading out of range registers.